src/OpenTelemetry.ResourceDetectors.Container/OpenTelemetry.ResourceDetectors.Container.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
If you decided modify this lines, maybe it is worth to switch it to
```suggestion
    <TargetFrameworks>$(NetMinimumSupportedVersion)</TargetFrameworks>
```

It will fully remove support for .NET Framework. As mentioned it does not have any benefits.


@iskiselev, what do you think?

Reply:
I like it, thank you for the advice.

Reply:
It will drop support of .Net prior to NET 6.0. OTEL does not officially support older, non-supported by MS .Net (Core) versions, but provides .Net Standard 2.0 library, that may be used in older .NET versions.
So, probably we would like to have
<TargetFrameworks>$(NetMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>

@Kielek, if support of older .NET Core version though Net Standard libraries was never a goal, I think it is OK to switch to just `<TargetFrameworks>$(NetMinimumSupportedVersion)</TargetFrameworks>`.

Reply:
IMO it is not a goal to support old versions. OTel SDK/API does nut support it. Transitively, we do not have this support.

Reply:
@anoopbabu29 Can you make this suggested change?

Reply:
Apologies for the delay, making it now